Tutorials/Tutorial25_StatePackager/src/Tutorial25_StatePackager.cpp

```diff
@@ -363,11 +354,23 @@ void Tutorial25_StatePackager::Render()
         m_pImmediateContext->Draw({3, DRAW_FLAG_VERIFY_ALL});
     }
 
+
+    const Uint32 CurrFrameIdx = (m_CurrentFrameNumber + 0) & 0x01;
+    const Uint32 PrevFrameIdx = (m_CurrentFrameNumber + 1) & 0x01;
+
+
     // Path trace
     {
         // Matches the THREAD_GROUP_SIZE in the render state notation file
         static constexpr Uint32 ThreadGroupSize = 8;
 
+        m_pPathTraceSRB->GetVariableByName(SHADER_TYPE_COMPUTE, "g_Albedo")->Set(m_GBuffer.pAlbedo->GetDefaultView(TEXTURE_VIEW_SHADER_RESOURCE));
+        m_pPathTraceSRB->GetVariableByName(SHADER_TYPE_COMPUTE, "g_Normal")->Set(m_GBuffer.pNormal->GetDefaultView(TEXTURE_VIEW_SHADER_RESOURCE));
+        m_pPathTraceSRB->GetVariableByName(SHADER_TYPE_COMPUTE, "g_Emittance")->Set(m_GBuffer.pEmittance->GetDefaultView(TEXTURE_VIEW_SHADER_RESOURCE));
+        m_pPathTraceSRB->GetVariableByName(SHADER_TYPE_COMPUTE, "g_Depth")->Set(m_GBuffer.pDepth->GetDefaultView(TEXTURE_VIEW_SHADER_RESOURCE));
+        m_pPathTraceSRB->GetVariableByName(SHADER_TYPE_COMPUTE, "g_RadianceInput")->Set(m_pRadianceAccumulationBuffer[PrevFrameIdx]->GetDefaultView(TEXTURE_VIEW_SHADER_RESOURCE));
+        m_pPathTraceSRB->GetVariableByName(SHADER_TYPE_COMPUTE, "g_RadianceOutput")->Set(m_pRadianceAccumulationBuffer[CurrFrameIdx]->GetDefaultView(TEXTURE_VIEW_UNORDERED_ACCESS));
```

Comment:
Instead of making everything DYNAMIC, create two SRBs for ping-ponging, but only on WebGPU